components/Layouts/Sidebar/SessionCard.tsx

```diff
@@ -1,29 +1,93 @@
-import { FC } from 'react';
+import { FC, useEffect, useRef, useState } from 'react';
 import { User } from '@prisma/client';
-import Link from 'next/link';
 import { signOut } from 'next-auth/react';
 import { IoLogOut } from 'react-icons/io5';
-import { MdPlaylistAdd } from 'react-icons/md';
+import { MdLogout } from 'react-icons/md';
+import Image from 'next/legacy/image';
+
+const getProfilePictureSrc = (name: string, image: string | null) => {
+  return (
+    image ||
+    `https://avatars.dicebear.com/api/initials/${name}.png?backgroundColorLevel=800&fontSize=40`
+  );
+};
+
+export const SessionCard: FC<Pick<User, 'email' | 'name' | 'image'>> = ({
+  name,
+  image,
+}) => {
+  const [isDropdownOpen, setIsDropdownOpen] = useState(false);
+  const dropdownRef = useRef<HTMLDivElement>(null);
+  const profilePictureSrc = getProfilePictureSrc(name as string, image);
+
+  const toggleDropdown = () => {
+    setIsDropdownOpen((prevState) => !prevState);
+  };
+
+  useEffect(() => {
+    const handleClickOutside = (event: MouseEvent) => {
+      if (
+        dropdownRef.current &&
+        !dropdownRef.current.contains(event.target as Node)
+      ) {
+        setIsDropdownOpen(false);
+      }
+    };
+
+    document.addEventListener('mousedown', handleClickOutside);
+
+    return () => {
+      document.removeEventListener('mousedown', handleClickOutside);
+    };
+  }, []);
 
-export const SessionCard: FC<Pick<User, 'email' | 'name' | 'image'>> = () => {
   return (
     <>
-      <Link href="/projects/add-project">
-        <li className="mt-2 h-9 cursor-pointer text-gray-400 transition-all hover:text-gray-300">
-          <MdPlaylistAdd size={28} className="block lg:hidden" />
-          <span className="hidden text-lg text-gray-400 hover:text-gray-300 lg:block ">
-            Add project
-          </span>
-        </li>
-      </Link>
       <li
-        onClick={() => signOut({ redirect: false })}
-        className="h-9 cursor-pointer text-gray-400 transition-all hover:text-gray-300"
+        onClick={toggleDropdown}
+        className="relative h-9 w-full cursor-pointer text-gray-400 transition-all hover:text-gray-300"
       >
-        <IoLogOut size={28} className="block lg:hidden" />
-        <span className="hidden text-lg text-red-400 transition-colors hover:text-red-500 lg:block ">
-          Logout
+        <div className="block flex justify-center lg:hidden">
```

Comment:
block and flex can't stay same time. remove `block
```suggestion
        <div className="flex justify-center lg:hidden">
```